File: STRIVER/video/vid-5/pattern4/p4.py
#  1
#  2 2
#  3 3 3
#  4 4 4 4


def p3(row:int)->None:
    for i in range(1,row+1):
        for j in range(1, i+1):
            print(i, end=" ")
        print("")

row = 5
p3(row)

File: STRIVER/video/vid-5/pattern4/p4.cpp
//  1
//  2 2
//  3 3 3
//  4 4 4 4

#include <iostream>
using namespace std;

int main()
{
    int row = 5;

    for (int i = 1; i <= row; i++)
    {
        for (int j = 1; j <= i; j++)
        {
            cout << i<<" ";
        }
        cout << endl;
    }
}